packages/expo-localization/src/Localization.ts

```diff
@@ -100,6 +144,8 @@ export const region = ExpoLocalization.region;
  *
  * const { locale } = await Localization.getLocalizationAsync();
  * ```
+ * @deprecated
+ * Use Localization.getLocales() or Localization.getCalendars() instead.
```

Comment:
```suggestion
 * @deprecated Use Localization.getLocales() or Localization.getCalendars() instead.
```
Not sure, but listing message after the `deprecated` annotation might produce incorrect appearance in docs.

Reply:
<img width="834" alt="image" src="https://user-images.githubusercontent.com/5597580/193564793-6ba50062-5feb-4185-abe5-61123ccb02ce.png">
It seems to produce the correct result regardless of the order